hs_access_control/migrations/0021_auto_20170418_2042.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
```

Comment:
Also try deleting this migration file and see if it works without this migration file.